src/NeoSharp.Core/Network/Rpc/Mvc/MvcRpcNormalizeRespMiddleware.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace NeoSharp.Core.Network.Rpc.Mvc
+{
+    public class MvcRpcNormalizeRespMiddleware
+    {
+        private readonly RequestDelegate _next;
+
+        public MvcRpcNormalizeRespMiddleware(RequestDelegate next)
+        {
+            _next = next;
+        }
+
+        public async Task Invoke(HttpContext context)
+        {
+            var originBody = context.Response.Body;
+            var newBody = new MemoryStream();
+            context.Response.Body = newBody;
+            await _next(context);
+            newBody.Seek(0, SeekOrigin.Begin);
+
+            JObject response = new JObject();
+            var resultStr = new StreamReader(newBody).ReadToEnd();
```

Comment:
We need a max-post check here (security)

Reply:
@melanke Please fix this before merge

Reply:
Done